### PR TITLE
Feat/home page

### PR DESCRIPTION
**Pull request by sonishreyas**
# Change Logs

- added Motivational quotes feature
- added slider in banner
- Header and footer added and made responsive
- NavBar added

#Deploy Preview
https://deploy-preview-1--ripple-habits.netlify.app/

### CODE REVIEW

File: src/components/Home/Banner.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+							className={`banner ${
+								slideIndex === index + 1 ? "slide-active" : "slide-inactive"
+							}`}
+							alt="banner"
+						/>
+					</>
+				))}
+			</article>
+			<div className="flex-row justify-content-center align-center flex-gap-half banner-dots-container">
+				{Array.from({ length: sliderData.length }).map((item, index) => (
```
Comment (LordUttam) [inline src/components/Home/Banner.jsx:48]: I did not know about `Array.from()`. Going to use it from now. :)

File: src/components/Home/Banner.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+	}, [slideIndex]);
+	return (
+		<>
+			<article className="background-image-container ">
+				{sliderData.map(({ id, gifURL }, index) => (
+					<>
+						<img
+							src={gifURL}
+							className={`banner ${
+								slideIndex === index + 1 ? "slide-active" : "slide-inactive"
```
Comment (LordUttam) [inline src/components/Home/Banner.jsx:40]: Everything looks far apart. Run prettier.

**Review by LordUttam — APPROVED** [review]
Run prettier. Rest looks good to me. Cool UI

**Review by yashpatel1797 — APPROVED** [review]
code looks clean
file structure is good
design looks consistent

**Pull request merged**